containerd/nslistener.go

```diff
@@ -83,6 +83,8 @@ func collectionInterfaceInfo() []supervisor.InterfaceInfo {
 			glog.Infof("get interface %v", info)
 			infos = append(infos, info)
 		}
+		// set link down, tap device take over it
+		netlink.LinkSetDown(link)
```

Comment:
does  the docker daemon try to retrieve any information in some cases when the container is running?